src/main/java/org/jenkinsci/plugins/registry/notification/DockerHubTrigger.java

```diff
@@ -69,6 +70,7 @@
 public class DockerHubTrigger extends Trigger<Job<?, ?>> {
 
     private List<TriggerOption> options;
+    private List<EventType> eventTypes = new Vector<EventType>();
```

Comment:
_nit_ Both the set and get handles `null` so no real reason to initialize it here.

Reply:
good point - will remove